### ISSUE TITLE
Not able to add EventProcessor package to dotnetcore project

### ISSUE THREAD

**Issue by niksacdev** [Azure/azure-event-hubs-dotnet#146]
## Actual Behavior
1. Microsoft.Azure.EventHubs.Processor package is failing with the below error:
Checking compatibility for Microsoft.Data.Edm 5.7.0 with .NETStandard,Version=v1.3.
Package Microsoft.Data.Edm 5.7.0 is not compatible with netstandard1.3 (.NETStandard,Version=v1.3). Package Microsoft.Data.Edm 5.7.0 supports:
  - net40 (.NETFramework,Version=v4.0)
  - portable-net40+sl5+win8+wp8+wpa81 (.NETPortable,Version=v0.0,Profile=Profile328)
  - portable-net45+win8+wp8+wpa81 (.NETPortable,Version=v0.0,Profile=Profile259)
  - sl4 (Silverlight,Version=v4.0)
2. I have tried moving my project to higher .NET standard version upto 1.6 and get the same error.

## Expected Behavior
1. I should be able to use EventProcessor in dotnetcore 

## Versions  
- OS platform and version: MacOS 10.12.3
- .NET Version: dotnetcore 1.1.1
- NuGet package version or commit ID: (latest stable)

**Comment by serkantkaraca** [comment]
Can you share your project.json?

**Comment by srmrco** [comment]
@serkantkaraca I have the same issue when using VS2017, I create an empty .net core console app and try to add the `Microsoft.Azure.EventHubs.Processor` nuget package.

Here is the full contents of my csproj file:

```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>netcoreapp1.1</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.Azure.EventHubs" Version="1.0.0" />
  </ItemGroup>

</Project>
```

As you can see `Microsoft.Azure.EventHubs` adds just fine but `Microsoft.Azure.EventHubs.Processor` does not. Any ideas?

**Comment by serkantkaraca** [comment]
Can you try updating 2017 project file regarding below. The default csproj created does not contain platform portability settings.

`<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>netcoreapp1.0</TargetFramework>
    <AssemblyName>ConsoleApp5</AssemblyName>
    <OutputType>Exe</OutputType>
    <PackageId>ConsoleApp5</PackageId>
    <RuntimeFrameworkVersion>1.1.1</RuntimeFrameworkVersion>
    <PackageTargetFallback>$(PackageTargetFallback);dnxcore50;portable-net45+win8</PackageTargetFallback>
    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
    <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
    <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.Azure.EventHubs" Version="1.0.0" />
    <PackageReference Include="Microsoft.Azure.EventHubs.Processor" Version="1.0.0" />
  </ItemGroup>

</Project>
`

**Comment by srmrco** [comment]
@serkantkaraca I worked, thank you!

**Comment by serkantkaraca** [comment]
One final update. It seems Storage has fixed build issues in 8.1.1 release. We will target to move to this new version hoping to address the package issue on 2017.
